build/release/post-release.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env sh
```

Comment:
This seems like overkill for `sh`, why not:
```suggestion
#!/bin/sh
```
?